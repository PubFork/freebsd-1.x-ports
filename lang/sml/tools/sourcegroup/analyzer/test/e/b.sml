
signature A = sig type t val f :unit-> t end;

functor F (AQ:A) = struct val x:AQ.t=AQ.f() end;

funsig FS (AP:A) = sig val x:AP.t end

(* b.sml:6.30-6.33 Error: unbound structure: AP in path AP.t *)

functor AA :FS = F

