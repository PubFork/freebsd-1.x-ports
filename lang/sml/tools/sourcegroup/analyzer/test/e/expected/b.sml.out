source sml "e/b.sml"
  export signature A 
  export functor AA F 
  export fsig FS ;

